src/test/java/com/wikia/webdriver/testcases/discussions/mobile/PostsListAnon.java

```diff
@@ -0,0 +1,24 @@
+package com.wikia.webdriver.testcases.discussions.mobile;
+
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.User;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.mercury.discussions.PostsListPage;
+
+import org.testng.annotations.Test;
+
+/**
+ * Created by qaga on 2015-11-24.
+ */
+@Test(groups = {"Mercury", "Discussions"})
```

Comment:
Don't add `Mercury` group, we will drop that group soon (for now it is not used)
Just use `@Test(groups = "Discussions")`